.github/workflows/fastcdr-test.yml

```diff
@@ -54,37 +56,37 @@ jobs:
           - 'ubuntu-20.04'
           - 'ubuntu-22.04'
           - 'windows-2019'
+          - 'macos-13'
 
     steps:
     - name: Sync eProsima/Fast-CDR repository
-      uses: eProsima/eProsima-CI/external/checkout@v0
+      uses: eProsima/eProsima-CI/external/checkout@feature/fastdds_mac_ci_support
```

Comment:
Just to ensure that before merging a release have been done and this tag updated.